client/packages/requisitions/src/RequestRequisition/DetailView/IndicatorEdit/IndicatorLineEdit.tsx

```diff
@@ -95,7 +95,7 @@ const InputWithLabel = ({
     <InputWithLabelRow
       Input={inputComponent}
       labelWidth={LABEL_WIDTH}
-      label={indicatorColumnNameToLocal(data.name)}
+      label={t(indicatorColumnNameToLocaleKey(data.name))}
```

Comment:
This looks like the one change that would fix the issue

Reply:
Actually `It does remove the fallback, but we only have comment and value so I think it's ok.`, we should pass `t` to `indicatorColumnNameToLocaleKey` and then use it there, so we can keep fallback

Reply:
I've changed to pass around the TypedTFunction